packages/jaeger-ui/src/components/TracePage/TraceTagOverview/PopupSQL.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2018 The Jaeger Authors.
```

Comment:
For consistency (e.g. with `UiFindInput.*`), the file should be named `PopupSql.css`